modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -3,9 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You need to have access to an API in your organization's Exchange to perform this procedure. For example, in Design Center create an API and publish it to Exchange. After setting up the API to manage it from Exchange, the API version, status, and other information appears in API Manager. With this set up, you can configure and manage the API.
+To be able to manage your API that you created using Design Center or any other application, you must first import it to Anypoint Exchange (Exchange). After you create the API in API Manager by importing it from Exchange, you can modify the API version, status, and other information. After saving the configuration, you can deploy the API.
```

Comment:
```suggestion
To manage an API that you create, either using Design Center or any other application, you must first import the API to Anypoint Exchange (Exchange). After you create the API in API Manager by importing it from Exchange, you can modify the API version, status, and other information. After saving the configuration, you can deploy the API.
```
This chronology is confusing:
You create an API.
To manage it, you import to Exchange.
Then you create it in API Manager by importing it from Exchange to manage it? You have already created it.
Where is the user working: in Exchange or API Manager?
Are you saying...:
1. Create an API
2. Import to Exchange.
3. Export to API Manager.
4. In API Manager, modify.
5. Save.
6. Deploy to ...where?

Reply:
Is this illustrated?

Reply:
Looking at the title (which should be fixed to use this capitalization: Create an API Instance by Import from Exchange), I think what you mean is this:
```suggestion
To manage the API that you created using Design Center or any another application, you must first import it to Anypoint Exchange (Exchange). Then you import the API to API Manager, from which you can modify the API version, status, and other information. After saving your changes, deploy the API.
```
Deploy to...?
Also, does this mean that the original API still exists in Exchange, and the modified one in API Manager?

Reply:
Done! Thanks!